app/test_main.py

```diff
@@ -1,3 +1,17 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+class TestHumanAge:
```

Comment:
i think u must implement 2 separate tests for dog and cat, cause it's not obvious where is error